express-hello-world/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res) {
+  res.send('Hello World');
+});
+
+app.listen(3000);
```

Comment:
Although not required it is still good to use the callback function for listen so you know the server started successfully and a reminder of the PORT number.